server/repository/src/migrations/mod.rs

```diff
@@ -60,6 +63,7 @@ pub fn migrate(
         Box::new(V1_01_01),
         Box::new(V1_01_02),
         Box::new(V1_01_03),
+        Box::new(v1_01_05::V1_01_05),
```

Comment:
A little bit less import when fully qualifying with, `v1_01_05::V1_01_05` vs `V1_01_05` and `use self::v1_01_05::V1_01_05`